x-pack/legacy/plugins/code/common/commit_utils.ts

```diff
@@ -4,6 +4,21 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import moment from 'moment';
+import { i18n } from '@kbn/i18n';
+
+const commitDateFormats: { [key: string]: string } = {
+  en: 'MMM Do, YYYY',
+  'zh-cn': 'YYYY年MoDo',
+};
+
+export const formatCommitDate: (dateString: string) => string = date => {
+  const locale = i18n.getLocale();
```

Comment:
just to learn, is this the recommended way from Kibana to handle Date rendering?

Reply:
oh, I just find that you moved it from one file below. so ignore my comment.